somaticseq.wdl

```diff
@@ -0,0 +1,289 @@
+version 1.0
+
+import "common.wdl" as common
+
+task SomaticSeqParallelPaired {
```

Comment:
This is already in a file called somaticseq.
So the `SomaticSeq` part of the task can be removed.
This can then be called as `somaticseq.ParallelPaired` in the workflow, which is clear enought.